### PR TITLE
Google maps minification

### PR DESCRIPTION
**Pull request by sai-manohar**

### CODE REVIEW

File: GoogleMaps/search-location.toprocess.html [hunk 1]
```diff
@@ -140,7 +146,14 @@ <h5 class="star-ratings-heading">{{vm.place.rating}}</h5>
...
   <script src="lib/ngmap/build/scripts/ng-map.min.js?ver=<!-- @echo VERSION -->"></script>
   <script src="lib/toastr/toastr.min.js?ver=<!-- @echo VERSION -->"></script>
-  <script src="dist/js/search-location.js?ver=<!-- @echo VERSION -->"></script>
+   <!-- @if NODE_ENV='production' -->
+   <script src="dist/js/search-location.min.js?ver=<!-- @echo VERSION -->"></script>
+   <!-- @endif -->
+
+   <!-- @if NODE_ENV='development' -->
+   <script src="dist/js/search-location.js?ver=<!-- @echo VERSION -->"></script>
+   <<!-- @endif -->
```
Comment (SirishaGrandhe) [inline GoogleMaps/search-location.toprocess.html]: @sai-manohar Please fix issue with extra `<` symbol

File: GoogleMaps/search-location.toprocess.html [hunk 2]
```diff
@@ -14,7 +14,13 @@
   <link rel="stylesheet" href="lib/bootstrap/dist/css/bootstrap.min.css?ver=<!-- @echo VERSION -->">
   <link rel="stylesheet" href="lib/toastr/toastr.css?ver=<!-- @echo VERSION -->">
   <link rel="stylesheet" href="lib/angular-material/angular-material.min.css?ver=<!-- @echo VERSION -->">
-  <link rel="stylesheet" href="dist/css/style.css?ver=<!-- @echo VERSION -->">
+   <!-- @if NODE_ENV='production' -->
+     <link rel="stylesheet" href="dist/css/style.min.css?ver=<!-- @echo VERSION -->">
```
Comment (SirishaGrandhe) [inline GoogleMaps/search-location.toprocess.html]: @sai-manohar Code is not indented properly.

File: GoogleMaps/index.toprocess.html [hunk 3]
```diff
@@ -82,8 +89,17 @@ <h5 class="modal-title">Google maps</h5>
...
   <script src="lib/ngmap/build/scripts/ng-map.min.js?ver=<!-- @echo VERSION -->"></script>
   <script src="lib/toastr/toastr.min.js?ver=<!-- @echo VERSION -->"></script>
+
+  <!-- @if NODE_ENV='production'-->
+  <script src="dist/js/bootstrap-contributors.min.js?ver=<!-- @echo VERSION -->"></script>
+  <!-- @endif -->
+
+  <!-- @if NODE_ENV='development' -->
   <script src="dist/js/bootstrap-contributors.js?ver=<!-- @echo VERSION -->"></script>
+  <<!-- @endif -->
```
Comment (SirishaGrandhe) [inline GoogleMaps/index.toprocess.html]: @sai-manohar Please fix the issue with extra `<` symbol.

File: GoogleMaps/Gruntfile.js [hunk 4]
```diff
@@ -36,11 +38,13 @@ module.exports = function (grunt) {
...
   minifyFiles[distPath + 'js/main.min.js'] = jsFiles;
   minifyFiles[distPath + 'js/initialScripts.min.js'] = initialLoadScripts;
+  minifyFiles[distPath + 'js/bootstrap-contributors.min.js'] = 'dist/js/bootstrap-contributors.js';
+  minifyFiles[distPath + 'js/search-location.min.js'] = 'dist/js/search-location.js';
 
   var preprocessOpts = {
     context: {
       DEBUG: true,
-      NODE_ENV: 'development',
+      NODE_ENV: 'production',
```
Comment (SirishaGrandhe) [inline GoogleMaps/Gruntfile.js]: @sai-manohar Add grunt options to change the environment dynamically.

Comment (sai-manohar) [reply GoogleMaps/Gruntfile.js]: Added code to check grunt options if given as params.

**Pull request merged**